config/locales/idv/en.yml

```diff
@@ -35,7 +35,6 @@ en:
       use_financial_account: Enter one of these here >
       zipcode: ZIP Code
     index:
-      cancel_link: No, I don’t have it
```

Comment:
My goal was to get them all consistent to start, and then if we determine we want to vary the content of the text in the future I can add another variable to that partial. Last I heard it should just be `Cancel` for now.